### PR TITLE
add aditya mukti as student

### PR DESCRIPTION
**Pull request by skywrapper**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by Elgayar** [comment]
Please resolve the conflicts, check the boxes and write a description so that we can merge!

**Review by abishekvashok — CHANGES_REQUESTED** [review]
@skywrapper nice to see you here.

Please resize your image (i meant your and not the blog one) to exactly 240x240 pixels yours is 241x238 pixels, pls rectify it and update the PR body after reading it thoroughly, besides that there was a conflict which however we are resolving for now but next time I want to see you resolving conflicts cause we want you to learn ;)

Happy coding, please fix the image so that we could merge.

**Comment by abishekvashok** [comment]
And if possible squash your commits

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Resize images.

**Comment by robbyoconnor** [comment]
@skywrapper please read the text in the text that was there when you opened it.

**Comment by robbyoconnor** [comment]
@skywrapper also your blog screenshot looks rather distorted..

File: _data/students.yml [hunk 1]
```diff
@@ -821,10 +821,19 @@
   lat: -8.6539591
   lng: 115.22582229999999
 
+- name: Aditya Mukti
+  github: skywrapper
+  image: aditya mukti.png
```
Comment (robbyoconnor) [inline _data/students.yml]: Image name cannot have spaces.

File: _data/students.yml [hunk 2]
```diff
@@ -821,10 +821,19 @@
...
+  facebook: Muhammad Aditya Mukti Nugroho
```
Comment (robbyoconnor) [inline _data/students.yml]: FB username cannot have spaces.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
- Add image
- Image name cannot contain spaces
- FB usernames also may not contain spaces.

**Comment by robbyoconnor** [comment]
Also resolve merge conflicts.

**Comment by robbyoconnor** [comment]
@skywrapper you around?

**Review by skywrapper — COMMENTED** [review]
i  change the mistake with space between my facebook name, i re upload my photo with thr right size. and change the my sutudent photo name whitput space.

**Comment by robbyoconnor** [comment]
@skywrapper 

- [Resolve the conflict](https://github.com/fossasia/gci16.fossasia.org/pull/852/conflicts)
- Your blog screenshot is skewed -- fix that.

File: _data/students.yml [hunk 3]
```diff
@@ -821,10 +821,19 @@
   lat: -8.6539591
   lng: 115.22582229999999
 
+- name: Aditya Mukti
+  github: skywrapper
+  image: adityamukti.png
+  facebook: MuhammadAdityaMuktiNugroho
```
Comment (robbyoconnor) [inline _data/students.yml]: There is no profile.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Facebook username is non-existent.

**Review by skywrapper — COMMENTED** [review]
i change my fb profile

**Review by skywrapper — COMMENTED** [review]
i change my photo blog

**Comment by abishekvashok** [comment]
Can you update the pr body

**Comment by abishekvashok** [comment]
@skywrapper please update the PR body!

**Comment by robbyoconnor** [comment]
@skywrapper please delete your fork and re-add yourself. I am going to close this pull request since it's now a disaster zone.

**Comment by robbyoconnor** [comment]
@skywrapper when you readd yourself -- read the first line of the file and add yourself to a random location.

**Comment by skywrapper** [comment]
sorry for the mistake, i am trying to squash the commit and push them. i
read the guide in someone blog, but i dont know it will push all the
commits.
but i will re join this task with other account in gci because i already
abandon the task in gci
thanks for the guide

2017-01-11 3:46 GMT+08:00 robbyoconnor <notifications@github.com>:

**Pull request closed without merging**